backend/person/models.py

```diff
@@ -22,6 +29,84 @@ def __str__(self):
         return self.name
 
 
+class HistoricalPerson(HistoricalEntity, models.Model):
+    """
+    A historical figure, which may be referenced in narrative sources or preserved letters
+    """
+
+    pass
+
+
+class PersonDateOfBirth(Field, LettercraftDate, models.Model):
+    person = models.OneToOneField(
+        to=HistoricalPerson,
```

Comment:
I wasn't really sure about this, it's true that it's not really consistent.

I think it makes sense to me because I'm not a historian but I have some background in queer theory, so I see dates as boring background information, and representations of gender as dynamic and interesting. That framing depends on research interest, though.